### PR TITLE
Shipping Manager

### PR DESCRIPTION
**Pull request by marcoskwkm**
#### What is the purpose of this pull request?
The time has come. This PR merges the Shipping Manager refactor branch into master and prepares for launch.

All commits (except for release and changelog commits) in the `refactor/shipping-manager` branch were already reviewed in [past PRs](https://github.com/vtex/omnishipping/pulls?q=is%3Apr+base%3Arefactor%2Fshipping-manager). There's no need to review all changes except for the changelog.

#### How should this be manually tested?
Use these workspaces ([lean](https://shpmgr2--vtexgame1.myvtex.com/checkout/cart/add?sku=289&qty=1&seller=1)/[nolean](https://shpmgr2--vtexgame1nolean.myvtex.com/checkout/cart/add?sku=289&qty=1&seller=1)), which have ***just omnishipping*** linked. I can't provide a specific test plan, so just test it with random items and see if everything is fine.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.
- [x] Refactor

### CODE REVIEW

**Comment by lucasecdb** [comment]
@marcoskwkm I'm seeing the following scenario in the nolean workspace:

image

[Cart link](https://shpmgr2--vtexgame1nolean.myvtex.com/checkout/cart/add/?sku=289&qty=1&seller=1&sc=1&sku=303&qty=1&seller=1&sc=1&sku=36&qty=1&seller=1&sc=1&sku=283&qty=1&seller=1&sc=1).

**Comment by marcoskwkm** [comment]
> @marcoskwkm I'm seeing the following scenario in the nolean workspace:

[Looking at the code](https://github.com/vtex/shipping-manager/blob/master/react/reducers/delivery-reducer.js#L101) it seems to be the intended behavior when the SKUs don't all have the same SLA array. In this case, SKU 289 has 4 SLAs and SKU 283 has only one. This is the behavior in `master` as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Big refactor to use Shipping Manager, which unifies the business logic of Omnishipping and Shipping Preview.
```
Comment (kevinch) [inline CHANGELOG.md:12]: Just a suggestion. I like pragmatism.
```suggestion
- Refactor to use Shipping Manager (unifies business logic of Omnishipping and Shipping Preview).
```

Comment (marcoskwkm) [reply CHANGELOG.md:12]: I'll take the suggestion of removing "big", but I'm a bit against the excessive use of parenthesis in sentences (despite me using those all the time, like right now).

**Pull request merged**